### PR TITLE
Archive artifacts

### PR DESCRIPTION
**Pull request by Katka92**
Trying to archive artifacts. Disabled usage of workspace due to openshift.io issue.

### CODE REVIEW

**Comment by Katka92** [comment]
Inspiration from Radim https://github.com/redhat-developer/che-functional-tests/pull/92

**Comment by Katka92** [comment]
@rhopp @ScrewTSW you can finally review it :)

File: cico/cico_run_EE_tests.sh [hunk 1]
```diff
@@ -1,4 +1,21 @@
 #!/bin/bash
+archive_artifacts(){
+  echo "Archiving artifacts"
+  DATE=$(date +%Y%m%d-%H%M%S)
+  echo "With date $DATE"
```
Comment (rhopp) [inline cico/cico_run_EE_tests.sh]: I guess these two lines are not necessary ;-)

File: cico/cico_run_EE_tests.sh [hunk 2]
```diff
@@ -1,4 +1,21 @@
...
+  chmod 600 ./artifacts.key
+  chown root ./artifacts.key
+  ls -lR ./tests/target/surefire-reports
+  ls -lR ./tests/target/screenshots
+  mkdir -p ./che-functional-tests/${JOB_NAME}/${BUILD_NUMBER}/surefire-reports
+  cp -R ./tests/target/surefire-reports/* ./che-functional-tests/${JOB_NAME}/${BUILD_NUMBER}/surefire-reports
+  mkdir -p ./che-functional-tests/${JOB_NAME}/${BUILD_NUMBER}/screenshots
+  cp -R ./tests/target/screenshots/* ./che-functional-tests/${JOB_NAME}/${BUILD_NUMBER}/screenshots
+  rsync --password-file=./artifacts.key -PHva --relative ./che-functional-tests/${JOB_NAME}/${BUILD_NUMBER}/surefire-reports devtools@artifacts.ci.centos.org::devtools/
+  rsync --password-file=./artifacts.key -PHva --relative ./che-functional-tests/${JOB_NAME}/${BUILD_NUMBER}/screenshots devtools@artifacts.ci.centos.org::devtools/
```
Comment (rhopp) [inline cico/cico_run_EE_tests.sh]: Is it really needed to have two `rsync`s? Wouldnt one `rsync` `./che-functional-tests/${JOB_NAME}/${BUILD_NUMBER}/` (without surefire-reports and screenshots) be enough?

File: cico/prepare_environment.sh [hunk 3]
```diff
@@ -48,7 +48,7 @@ set +x
 
 #prepend "export " and remove space after "="
 cat jenkins-env \
-    | grep -E "(OSIO|KEYCLOAK)" \
+    | grep -E "(OSIO|KEYCLOAK|BUILD_NUMBER|JOB_NAME)" \
```
Comment (rhopp) [inline cico/prepare_environment.sh]: I may be nitpicking, but you are putting BUILD_NUMBER and JOB_NAME into file named `credential_file`. Just sayin' 😄

**Comment by rhopp** [comment]
Superseded by https://github.com/redhat-developer/che-functional-tests/pull/159
Closing.

**Pull request closed without merging**